src/protocol/commands/_qXfer_libraries_svr4_read.rs

```diff
@@ -0,0 +1,18 @@
+use crate::protocol::common::qxfer::ParseAnnex;
+use crate::protocol::common::qxfer::QXferReadBase;
+
+pub type qXferLibrariesSvr4Read<'a> = QXferReadBase<'a, LibrariesSvr4Annex>;
+
+#[derive(Debug)]
+pub struct LibrariesSvr4Annex;
+
+impl<'a> ParseAnnex<'a> for LibrariesSvr4Annex {
+    #[inline(always)]
+    fn from_buf(buf: &[u8]) -> Option<Self> {
+        if buf != b"" {
```

Comment:
meta: seems like the `qXfer:libraries-svr4:read` packet also supports an optional "augmented form"?

> If the remote stub indicates it supports the augmented form of this packet then the annex part of the generic ‘qXfer’ packet may contain a semicolon-separated list of ‘name=value’ arguments. The currently supported arguments are:

> start=address
>A hexadecimal number specifying the address of the ‘struct link_map’ to start reading the library list from. If unset or zero then the first ‘struct link_map’ in the library list will be chosen as the starting point.

>prev=address
>A hexadecimal number specifying the address of the ‘struct link_map’ immediately preceding the ‘struct link_map’ specified by the ‘start’ argument. If unset or zero then the remote stub will expect that no ‘struct link_map’ exists prior to the starting point.

>lmid=lmid
>A hexadecimal number specifying a namespace identifier. This is currently only used together with ‘start’ to provide the namespace identifier back to GDB in the response. GDB will only provide values that were previously reported to it. If unset, the response will include ‘lmid="0x0"’.

We don't have to support this as part of this PR, but if this looks like something that could be useful to support off-the-bat, I don't think it'd be _too_ hard to add.

Reply:
Oh good point! I haven't gotten up to speed on the gdb protocol here, but I'd be happy to poke arond and see what it might take to implement these for this PR

Reply:
It seems that the augmented form is purely additive, so it's totally something that we could land later as well.

Let me know if you'd like to take a crack at it while we're already here, or if you'd rather just land this PR as-is. Either way is OK with me